x-pack/legacy/plugins/siem/public/components/inspect/modal.tsx

```diff
@@ -39,6 +70,30 @@ const MyEuiModal = styled(EuiModal)`
   }
 `;
 
+const parseRequest = (objectStringify: string): Request | null => {
+  try {
+    return JSON.parse(objectStringify);
+  } catch {
+    return null;
+  }
+};
+
+const parseResponse = (objectStringify: string): Response | null => {
+  try {
+    return JSON.parse(objectStringify);
+  } catch {
+    return null;
+  }
+};
+
```

Comment:
I was curious if we can use the same parser for Request and Response, and found that this seems working well.
```
const parseInspectString = function<T>(objectStringify: string): T | null {
  try {
    return JSON.parse(objectStringify);
  } catch {
    return null;
  }
};
```